scripts/tools/DeliveryExecutionArchive.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\taoDeliveryRdf\scripts\tools;
+
+use common_report_Report as Report;
+use oat\oatbox\action\ResolutionException;
+use oat\oatbox\extension\AbstractAction;
+use oat\taoDelivery\model\DeliverArchiveExistingException;
+use oat\taoDelivery\model\DeliveryArchiveNotExistingException;
+use oat\taoDeliveryRdf\model\DeliveryArchiveService;
+
+/**
+ * Run examples:
+ *
+ * - Show list of deliveries:
+ * ```
+ * sudo -u www-data php index.php 'oat\taoDeliveryRdf\scripts\tools\DeliveryExecutionArchive' list
+ * ```
+ */
+class DeliveryExecutionArchive extends AbstractAction
```

Comment:
You should use the ontologyAwareTrait to access rdf class constructor.
You should use the constant \oat\taoDeliveryRdf\model\DeliveryAssemblyService::CLASS_URI for delivery uri
You should use service manager to fetch deliveryArchive service

Reply:
Right, I will adjust